CMakeLists.txt

```diff
@@ -446,6 +445,10 @@ add_subdirectory(${FLATBUFFERS_SRC_DIR}
         ${CMAKE_CURRENT_BINARY_DIR}/flatbuffers-build
         EXCLUDE_FROM_ALL)
 
+if (MSVC)
```

Comment:
https://stackoverflow.com/questions/70898030/boost-link-error-using-conan-find-package

Reply:
Let's add this to the file too, for future reference.